gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,10 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.L1A_ID")
+                                                                                          
+def getLinkL1Acount(device,link):
+    return readRegister(device, "GEM_AMC.DAQ.OH%s.COUNTERS.EVN"%(link))
+
```

Comment:
These are not probably what you want.
* Do you want a count of the number of triggers sent to the VFATs?
  * `GEM_AMC.OH.OH0.COUNTERS.T1.SENT.L1A`
* Do you want the number of events built from VFAT data sent to the AMC?
  * Then what you have will do this, but not all running modes will enable the DAQ link and event building, so this is probably the wrong way to count.